api/defaults.go

```diff
@@ -290,7 +290,6 @@ func defaultNodeConfig(installationID string, request *requests.CreateAccount, o
 	}
 
 	nodeConfig.Name = DefaultNodeName
-	nodeConfig.Rendezvous = false
```

Comment:
I was thinking that when we reintroduce rendezvous, it would be a good moment to move the config from being a value in the struct `NodeConfig`, to be a value in `WakuV2Config` similar to the `EnableDiscV5` attr.. The `Rendezvous` and `NoDiscovery` attributes living in the `NodeConfig` struct is too random hence my decision to remove it from there. (I'll remove `NoDiscovery` too in a separate PR)